schemast/field.go

```diff
@@ -197,6 +199,13 @@ func defaultExpr(d interface{}) (ast.Expr, error) {
 			Value: strconv.FormatBool(d.(bool)),
 		}
 		return lit, nil
+	case reflect.Func:
+		f := runtime.FuncForPC(v.Pointer()).Name()
+		parts := strings.Split(f, ".")
+		if len(parts) != 2 {
+			return nil, errors.New("schemast: only selector exprs are supported for default func")
+		}
```

Comment:
Please explain why?

Reply:
consider that you have a field
```go
field.Time("time").Default(func() time.Time { /.../})
```
At runtime, the field descriptor holds a reference to a function, it cannot build that functions AST.
If that function is declared in an external package, and referenced by a selector expression, for example `time.Now`, we can retain that reference and things should work fine when rewriting (because running `goimports` can fix the import for the user).

Reply:
We can also support `field.Time` and not support `Default(func)` at all

Reply:
We can support this by using pointers to the AST file and copy the string content as is, right?